CONTRIBUTING.md

```diff
@@ -132,34 +197,31 @@ Now navigate to your browser and open
 congratulations – you're all set. Otherwise, let us know by opening a GitHub
 issue and with your error.
 
-### Linting Setup
+### Make Changes
+This bit is up to you!
 
-You should have [ESLint running in your
-editor](http://eslint.org/docs/user-guide/integrations.html), and it will
-highlight anything doesn't conform to [Free Code Camp's JavaScript Style
-Guide](http://forum.freecodecamp.com/t/free-code-camp-javascript-style-guide/19121)
-(you can find a summary of those rules
-[here](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/.eslintrc).
-Please do not ignore any linting errors, as they are meant to **help** you and
-to ensure a clean and simple code base. Make sure none of your JavaScript is
-longer than 80 characters per line.  The reason we enforce this is because one
-of our dependent NPM modules, [jsonlint](https://github.com/zaach/jsonlint),
-does not fully support wildcard paths in Windows.
+### Run The Test Suite
+When your are ready to share your code, run the test suite:
 
-### Found a bug?
+```shell
+$ npm test
+```
+
+and ensure all tests pass. For Windows contributors, skip the jsonlint pretest by using this instead:
 
-Do not file an issue until you have followed these steps:
+```shell
+npm run test-challenges
+```
+This is because jsonlint will always fail on Windows, due to the wildcard parameters.
 
-1.  Read [Help I've Found a
-    Bug](http://forum.freecodecamp.com/t/how-to-report-a-bug/19543)
-    wiki page and follow the instructions there.
+### Squash Your Commits
+When you make a pull request, all of your changes need to be in one commit.
 
-2.  Asked for confirmation in the appropriate [Help Room](http://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390/2)
+If you have made more then one commit, then you will need to _squash_ your commits.
 
-3.  Please _do not_ open an issue without a 3rd party confirmation of your
-    problem.
+To do this, see [Rebasing](http://forum.freecodecamp.com/t/how-to-use-git-rebase/13226).
```

Comment:
I believe this should be `To do this, see [Squashing Your Commits](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231).`